resources/ts/Components/Dialog.tsx

```diff
@@ -36,7 +37,12 @@ export const Dialog = ({
           className="text-publiq-blue-dark self-end"
         />
 
-        <div className="flex flex-col flex-1 w-full p-4 text-xl">
+        <div
+          className={classNames(
+            "flex flex-col flex-1 w-full p-4 text-xl",
+            className
```

Comment:
It's misleading that className is not used in the root of this component.
You could introduce a prop `contentStyles` or something similar and pass it here